src/agent/mock-test-runner.js

```diff
@@ -59,6 +59,7 @@ export class MockTestRunner {
     return {
       command: command.id,
       expectation: assertion.expectation,
+      output: assertion.expectation,
```

Comment:
Let's make this value distinct so that the tests demonstrate we're not mishandling the `expectation` value, e.g. `mocked output for assertion "${assertion.expectation}"`